instagram-challenge/app/views/users/new.html.erb

```diff
@@ -0,0 +1,17 @@
+
+  <h1>Sign up</h1>
+   
+  <div class="form">
```

Comment:
this seems a bit redundant as the form generates a form tag. You can pass a class onto the form as an argument with  `class: 'whatever-class-you-want'`